pkg/core/native/std.go

```diff
@@ -160,8 +160,8 @@ func newStd() *Std {
 	return s
 }
 
-func (s *Std) serialize(_ *interop.Context, args []stackitem.Item) stackitem.Item {
-	data, err := stackitem.Serialize(args[0])
+func (s *Std) serialize(ic *interop.Context, args []stackitem.Item) stackitem.Item {
+	data, err := ic.DAO.GetItemCtx().Serialize(args[0], false)
```

Comment:
After that we create ByteArray right from `data` without copying. Are you sure it's not dangerous? `stackitem.NewByteArray` returns a pointer to the provided array. We may reuse the same serialisation context during the subsequent instructions processing, so I'm afraid the resulting byte array may be corrupted.

Reply:
Fixed.